youtube_dl/extractor/pornhub.py

```diff
@@ -281,20 +281,31 @@ def parse_js_value(inp):
 
         def add_video_url(video_url):
             v_url = url_or_none(video_url)
-            if not v_url:
+            if not v_url or not isinstance(v_url, compat_str):
                 return
             if v_url in video_urls_set:
                 return
             video_urls.append((v_url, None))
             video_urls_set.add(v_url)
 
+        def parse_quality_items(js_str):
+            media_definitions = self._parse_json(js_str, None)
+            if isinstance(media_definitions, list):
+                for definition in media_definitions:
+                    if not isinstance(definition, dict):
+                        continue
+                    video_url = definition.get('url')
```

Comment:
Inline.

Reply:
??

Reply:
?

Reply:
What do you mean by inline. Am I to change the for loop to a list comprehension??

Reply:
Inline the variable to the place of usage. What's not clear?